PlayerCollection.cs

```diff
@@ -101,6 +101,14 @@ public void ClearFormerPlayers()
             }
         }
 
+        public void SwapIcons()
```

Comment:
No need to have multiple names for the same action!

```suggestion
        public void RefreshIcons()
```